### ISSUE TITLE
The restore windows are very confusing

### ISSUE THREAD

**Issue by CottonEaster** [teejee2008/timeshift#255]
I've just done my first restore with TimeShift. I am glad to say it worked. Yet, the instructions are really not good, I am afraid.

Let's consider this first: 
![timeshift](https://user-images.githubusercontent.com/9998846/42845541-2e49d7c4-8a0e-11e8-9530-71685c1603a7.png)

'[W]here the files will be restored' would be clearer, and better English, were it to be: 'to which the files are to be restored.' Also, By 'devices', do you actually mean 'places', or something like that? For one need not, and typically will not, be restoring from or two multiple devices. Also, what the line about 'pre-selection' means is opaque.

Next, the 'Confirm actions' window. What is the box at the top? Need one fill it in? (Apparently not.)

Why does it say 'restore' under the 'location' field title? 'Restore' is not a location! Should this field be entitled 'action' instead?

I got the impression I could toggle individual restores and what the window calls 'creations'. Yet, I cannot.

Instructions for a restore process need to be _clear_. Your otherwise marvellous software is let down greatly by its text.

**Comment by groovenectar** [comment]
I agree, though I'm not sure what the wording should be changed to because it's confusing. I think that `to which the files are to be restored` might be even more confusing.

That said, it works and works well, but because of the confusing screen it kind of always feels like it worked by a stroke of luck...
